tasks/skylab/FastqProcessing.wdl

```diff
@@ -235,17 +235,18 @@ task FastqProcessingSlidSeq {
 task FastqProcessATAC {
 
     input {
-        Array[File] read1_fastq
-        Array[File] read3_fastq
-        Array[File] barcodes_fastq
+        Array[String] read1_fastq
+        Array[String] read3_fastq
+        Array[String] barcodes_fastq
         String read_structure = "16C"
         String barcode_orientation = "FIRST_BP_RC"
         String output_base_name
         File whitelist
 
         # [?] copied from corresponding optimus wdl for fastqprocessing
         # using the latest build of warp-tools in GCR
-        String docker = "us.gcr.io/broad-gotc-prod/warp-tools:1.0.1-1682971351"
+        # needs to change
+        String docker = "us.gcr.io/broad-gotc-prod/warp-tools:aa-gcloud-docker"
```

Comment:
If this works, we'll want to change this to the docker listed on the docker_versions tsv.